presto-hive/src/main/java/com/facebook/presto/hive/authentication/KerberosHiveMetastoreAuthentication.java

```diff
@@ -44,6 +45,28 @@
     public KerberosHiveMetastoreAuthentication(HiveClientConfig hiveClientConfig, @ForHiveMetastore HadoopAuthentication authentication)
     {
         this(hiveClientConfig.getHiveMetastoreServicePrincipal(), authentication);
+        verifyKerberosProperties(hiveClientConfig);
```

Comment:
I think we should be adding similar checks for configs in KerberosConfig.java. Those were the ones which had names similar to the airlift config properties.